File: src/main/java/com/lkimilhol/healingwords/words/domain/Words.java
package com.lkimilhol.healingwords.words.domain;

import javax.persistence.Embedded;
import javax.persistence.Entity;
import javax.persistence.Id;

import lombok.NoArgsConstructor;

import words.domain.Text;
import words.domain.Writer;

@Entity
@NoArgsConstructor
public class Words {

    @Id
    private Long id;

    @Embedded
    private Text text;

    @Embedded
    private Writer writer;

    // TODO 비밀번호 작성

    // TODO 작성일시

    // TODO 수정일시
}

File: src/main/kotlin/com/lkimilhol/healingwords/words/domain/Words.kt
package com.lkimilhol.healingwords.words.domain

import com.lkimilhol.healingwords.jpa.BaseEntity
import com.lkimilhol.healingwords.writer.domain.Writer
import javax.persistence.*

@Entity
@Table(name = "words")
class Words (
    @ManyToOne
    var writer: Writer?,

    @Embedded
    var text: Text?
) : BaseEntity() {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private var id: Long? = null
}